app/facades/food_db_facade.rb

```diff
@@ -0,0 +1,14 @@
+class FoodDBFacade
```

Comment:
This is a nicely structured facade! To minimize the number of API calls, you could do everything in one method and extract all the data you need from just one call. Consider that Ruby allows you to return more than one object from a method so you could, in your searched_food_results method, return:

`[all_results[0..9], parsed_json[:totalHits]`

In week 5, we'll also have a class on caching which would allow you to keep the organization of 2 different methods (which is nice and clean) and retrieve the totalHits data from the cache rather than making another API call